<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveDialog</class>
 <widget class="QDialog" name="SaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save file</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SaveDialog {
	background: url(:/background.jpg);
}

QPushButton { 
	border-radius: 5px;	
	background-color: rgb(12, 71, 24);
	border-style:outset;
	border-color: rgb(0, 90, 75);
	color: rgb(255, 255, 255);	
	font: 14pt ;
}

QPushButton:hover { 
	
	
	background-color: rgb(150, 224, 166);
	border-style: outset;
	border-color: rgb(0, 90, 75);
	border-width: 1px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(131, 182, 135);
border-radius: 5px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 14pt;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>FILE NAME:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filenameEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(131, 182, 135);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 14pt;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>.TXT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveButton">
        <property name="styleSheet">
         <string notr="true">min-width: 4.5em;</string>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
